cmd/sqlconfig/sqlconfig.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package sqlconfig
+
+type EndpointDetails struct {
+	Address string `mapstructure:"address"`
+	Port    int    `mapstructure:"port"`
+}
+
+type ContainerDetails struct {
+	Id    string `mapstructure:"id"`
+	Image string `mapstructure:"image"`
+}
+
+type AssetDetails struct {
```

Comment:
>t { [](http://example.com/codeflow?start=23&length=3)

where does one find instructions for using `mapstructure` and `yaml` attributes?

Reply:
They're called "struct tags", and you're right to ask, because they are barely documented in the golang spec:

This answers to this stackoverflow question seems to do the best at gathering the links to learn more:
 
[https://stackoverflow.com/questions/10858787/what-are-the-uses-for-struct-tags-in-go](https://stackoverflow.com/questions/10858787/what-are-the-uses-for-struct-tags-in-go)